src/digest/sha2.rs

```diff
@@ -19,23 +19,27 @@ use core::{
 };
 
 #[cfg(not(any(target_arch = "aarch64", target_arch = "arm", target_arch = "x86_64")))]
-pub(super) extern "C" fn GFp_sha256_block_data_order(
-    state: &mut super::State,
-    data: *const u8,
-    num: c::size_t,
-) {
-    let state = unsafe { &mut state.as32 };
-    *state = block_data_order(*state, data, num)
+versioned_extern_def! {
```

Comment:
Similar question here: If this isn't `#[no_mangle]` then why does it need `versioned_extern_def!`?